study-builder/studies/mpc/study-pdfs.conf

```diff
@@ -0,0 +1,176 @@
+{
+  "pdfs": [
+    {
+      "name": "mpcproject-consent",
+      "filename": "mpcproject-consent",
+      "displayName": "MPC consent pdf"
+      "mapping": {
+        "type": "CONSENT"
+      },
+      "versions": [
+        {
+          "tag": "v2",
+          "start": "2019-05-15T17:00:00Z",
```

Comment:
Did we track down that this was the official time when we deploy consent v2 to production?